src/main/java/com/project/model/Comment.java

```diff
@@ -0,0 +1,57 @@
+package com.project.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Data
+@Builder
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "application_comments")
+public class Comment {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "comment")
+    private String comment;
+
+    @Column(name = "comment_date")
+    private String commentDate;
+
+    @Column(name = "date_modified")
+    private String dateModified;
+
+    @Column(name = "is_visible_to_student")
+    private boolean isVisibleToApplicant;
+
+    @Column(name = "attachment_name")
+    private String attachmentName;
```

Comment:
Since the project is about to end at least for future reference I would suggest storing attachment information on a separate table. Your table objects are getting bigger and bigger as you add more data. Try to categorize your data and see what needs a separate table and what data should belong to which data model you have designed.